<?xml version="1.0"?>
<launch>
  <arg name="cam_name" default="csi_cam_0" />
  <arg name="topic_name" default="color_tracking" />

  <node pkg="jetracer" type="color_tracking.py" name="color_tracking" output="screen">
        <param name="camera_name"  value="$(arg cam_name)" />
	<param name="topic_name"  value="$(arg topic_name)" />
   </node>

</launch>

